web/src/components/PageHeader.tsx

```diff
@@ -60,15 +60,13 @@ export const PageHeader: React.FunctionComponent<Props> = ({ title }: Props) =>
           </a>
           <div className={classes.title}>{title}</div>
         </div>
-        <a
-          className={classes.contact}
-          href={FIDER_LINK}
-          target="_blank"
-          rel="noopener noreferrer"
+        <div
           id="contact-link"
+          className={classes.contact}
+          onClick={() => window.open('mailto:bcws.predictiveservice@gov.bc.ca')}
```

Comment:
```suggestion
          onClick={() => window.open('mailto:bcws.predictiveservices@gov.bc.ca')}
```

Reply:
Ah I blame Sybrand for not having a consistent mail address in the ticket, nice catch!

Reply:
it's a good approach, always blaming @Sybrand . I hold him responsible for many of my failings and inadequacies

Reply:
Always happy to take the blame.